CV/CV.adoc

```diff
@@ -0,0 +1,94 @@
+image:IMG_20200215_112749.jpg[IMG_20200215_112749,100,60,role=right]
+
+== Karolina Joanna Osuch +
```

Comment:
With asciidoctor export your cv to the html format and put it in the main folder of your project repo "Data_Analysis" folder